t/e2e/whisper/whisper_mailbox_test.go

```diff
@@ -276,6 +276,31 @@ func (s *WhisperMailboxSuite) TestRequestMessagesInGroupChat() {
 	s.Require().Equal(helloWorldMessage, messages[0]["payload"].(string))
 }
 
+func (s *WhisperMailboxSuite) TestSendMessageWithoutSubscription() {
+	aliceBackend, stop := s.startBackend("alice")
+	defer stop()
+
+	// we need to wain >= whisper.DefaultSyncAllowance seconds to update
+	time.Sleep(12 * time.Second)
+
+	//get whisper service
+	aliceWhisperService, err := aliceBackend.NodeManager().WhisperService()
+	s.Require().NoError(err)
+	//get rpc client
+	aliceRPCClient := aliceBackend.NodeManager().RPCClient()
+
+	//generate group chat symkey and topic
+	groupChatKeyID, err := aliceWhisperService.GenerateSymKey()
+	s.Require().NoError(err)
+	//generate group chat topic
+	groupChatTopic := whisper.BytesToTopic([]byte("groupChatTopic"))
+
+	//alice send message to group chat
+	helloWorldMessage := hexutil.Encode([]byte("Hello world!"))
+	s.postMessageToGroup(aliceRPCClient, groupChatKeyID, groupChatTopic.String(), helloWorldMessage)
+	time.Sleep(5 * time.Second) //it need to receive envelopes by bob and charlie nodes
```

Comment:
What happens after this waiting? No more asserts then?

Reply:
Nice catch!